src/test/java/entities/PlayerTest.java

```diff
@@ -0,0 +1,110 @@
+package entities;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PlayerTest {
+
+    Game game;
```

Comment:
Add access modifiers to class attributes and methods

Reply:
Fixed.